ui/hooks/useNftCollectionsMetadata.test.ts

```diff
@@ -0,0 +1,90 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { NameType } from '@metamask/name-controller';
+import { getCurrentChainId } from '../selectors';
+import { fetchNftCollectionsMetadata } from '../store/actions';
+import { useNftCollectionsMetadata } from './useNftCollectionsMetadata';
+
+jest.mock('react-redux', () => ({
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  useSelector: (selector: any) => selector(),
+}));
+
+jest.mock('../selectors', () => ({
+  getCurrentChainId: jest.fn(),
+}));
+
+jest.mock('../store/actions', () => ({
+  fetchNftCollectionsMetadata: jest.fn(),
+}));
+
+const PUNKS_ADDRESS = '0xb47e3cd837ddf8e4c57f05d70ab865de6e193bbb';
```

Comment:
Should we avoid naming things based on their value, and instead focus on the conceptual such as `NFT_CONTRACT_ADDRESS_MOCK_1` etc?

Reply:
Right, removed in https://github.com/MetaMask/metamask-extension/pull/25692/commits/eeda4a71b6b8b6849586ed226630829c42948c0e